src/components/network/constants.js

```diff
@@ -24,6 +24,11 @@ export const ENERGY_SOURCES = [
     { id: 'OTHER', label: 'Other' },
 ];
 
+export const REGULATION_TYPES = [
+    { id: 'DISTANT', label: 'Distant' },
```

Comment:
Maybe modify definition as it is made in REGULATION_MODES

Reply:
Done